src/Data/Function.purs

```diff
@@ -77,6 +81,20 @@ applyFlipped x f = f x
 -- | ```
 infixl 1 applyFlipped as #
 
+-- | Applies the function `f` to its argument `n` times.
```

Comment:
The thing about this is that, while it might be clear what `f` and `n` are to someone looking at the source, it's not really as clear if you're looking at the docs on Pursuit. You kind of have to be familiar with the convention that `n` is for naturals and `f` is for functions, which is often safe to assume, but probably best not to in the Prelude. This is why I'd like to say something like "the function `applyN f n` applies the function `f` ..." so that it's clear what `f` and `n` are. It doesn't have to be that verbatim, but I would like `f` and `n` to be introduced a little more explicitly.